products/drush.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Drush
+permalink: /drush
+releasePolicyLink: https://www.drush.org/latest/install/#drupal-compatibility
+category: app
+changelogTemplate: https://github.com/drush-ops/drush/releases/tag/__LATEST__
+activeSupportColumn: false
+releaseDateColumn: true
+releaseColumn: true
+eolColumn: Support
+versionCommand: drush --version
+auto:
+-   git: https://github.com/drush-ops/drush.git
+sortReleasesBy: 'releaseCycle'
+releases:
+-   releaseCycle: "11"
+    eol: false
+    latest: "11.0.9"
+    latestReleaseDate: 2022-04-16
+    releaseDate: 2022-01-11
+-   releaseCycle: "10"
+    eol: 2022-01-01
+    latest: "10.6.2"
+    latestReleaseDate: 2021-12-15
+    releaseDate: 2019-10-31
+-   releaseCycle: "9"
+    eol: 2020-05-01
+    latest: "9.7.3"
+    latestReleaseDate: 2021-03-22
+    releaseDate: 2018-01-24
+-   releaseCycle: "8"
+    eol: false
+    latest: "8.4.11"
+    latestReleaseDate: 2022-06-02
+    releaseDate: 2015-11-19
+-   releaseCycle: "7"
+    eol: 2017-07-01
+    latest: "7.4.0"
+    latestReleaseDate: 2022-03-25
+    releaseDate: 2016-09-22
+-   releaseCycle: "6"
+    eol: 2015-12-01
+    latest: "6.7.0"
+    latestReleaseDate: 2015-12-02
+    releaseDate: 2013-07-17
+-   releaseCycle: "5"
+    eol: 2015-05-01
+    latest: "5.11.0"
+    latestReleaseDate: 2014-05-22
+    releaseDate: 2012-03-23
+
+---
+
+> [Drush](https://www.drush.org/) is a command line shell and Unix scripting interface for Drupal.
+
+**Drush 11** is supported (for Drupal 9+).
```

Comment:
Reporting compatibility for unsupported versions doesn't add much. Neither does providing dates that are already in the table above. I'd prefer a 2 entry table like:

Drush|Supported PHP Versions|Supported Drupal Versions
---|---|---
11                  | 7.4+ | 9+
8 | 5.4.5+  | 7

(Dropping unsupported Drupal/Drush versions from the original table)